pybikes/data/gbfs.json

```diff
@@ -0,0 +1,523 @@
+                "name": "Nice Ride Minnesota",
+                "country": "US"
+            },
+            "feed_url": "https://api-core.niceridemn.org/gbfs/gbfs.json"
+        },
+        {
+            "tag": "pronto",
+            "meta": {
+                "latitude": 47.618418,
+                "longitude": -122.350964,
+                "city": "Seattle, WA",
+                "name": "Pronto",
+                "country": "US"
+            },
+            "feed_url": "https://gbfs.prontocycleshare.com/gbfs/gbfs.json"
+        },
+        {
+            "tag": "bcycle_sanantonio",
+            "meta": {
+                "latitude": 29.43701,
+                "longitude": -98.48236,
+                "city": "San Antonio, TX",
+                "name": "San Antonio B-cycle",
+                "country": "US"
+            },
+            "feed_url": "https://gbfs.bcycle.com/bcycle_sanantonio/gbfs.json"
+        },
+        {
+            "tag": "usf_share-a-bull_bikes",
+            "meta": {
+                "latitude": 28.060548696070995,
+                "longitude": -82.4076198041439,
+                "city": "University of South Florida",
+                "name": "Share-a-Bull Bikes",
+                "country": "US"
+            },
+            "feed_url": "https://usf.socialbicycles.com/opendata/gbfs.json"
+        },
+        {
+            "tag": "sobi_long_beach",
+            "meta": {
+                "latitude": 40.5833058203002,
+                "longitude": -73.6482682794673,
+                "city": "Long Beach, NY",
+                "name": "Sobi Long Beach",
+                "country": "US"
+            },
+            "feed_url": "https://sobilongbeach.com/opendata/gbfs.json"
+        },
+        {
+            "tag": "bcycle_spartanburg",
+            "meta": {
+                "latitude": 34.94867,
+                "longitude": -81.93345,
+                "city": "Spartanburg, SC",
+                "name": "Spartanburg BCycle",
+                "country": "US"
+            },
+            "feed_url": "https://gbfs.bcycle.com/bcycle_spartanburg/gbfs.json"
+        },
+        {
+            "tag": "topeka_metro_bikes",
+            "meta": {
+                "latitude": 39.0337646098385,
+                "longitude": -95.7027213202284,
+                "city": "Topeka, KS",
+                "name": "Topeka Metro Bikes",
+                "country": "US"
+            },
+            "feed_url": "http://topekametrobikes.org/opendata/gbfs.json"
+        },
+        {
+            "tag": "university_of_virginia",
+            "meta": {
+                "latitude": 38.0357625021372,
+                "longitude": -78.5072994363201,
+                "city": "University of Virginia",
+                "name": "Ubike",
+                "country": "US"
+            },
+            "feed_url": "https://ubike.virginia.edu/opendata/gbfs.json"
+        }
+    ],
+    "system": "gbfs",
+    "class": "Gbfs"
+}
```

Comment:
Noticed some networks are changing tags. That's ATM a bad idea, since system relies on networks being consistent (ie: someone consuming `sanantonio` feed will find out that it does not exist anymore and that it is now `bcycle_sanantonio`). There is no system in there currently to properly redirect or record network id changes except done manually on the API side. I know this is a boring task, so I will send a PR to this one.

Some notes about style:
- Add whitespace around operators (`i=10` -> `i = 10`)
- Block comments always start with `# `
- Do not use `\` for multiline expressions

For instance

```
        #Station Information and Station Status data retrieval
        station_information=json.loads(scraper.request(feeds['station_information']))\
            ['data']['stations']
```

Might become
```
        # Station Information and Station Status data retrieval
        station_information = json.loads(
            scraper.request(feeds['station_information'])
        )['data']['stations']
```

More info at https://www.python.org/dev/peps/pep-0008/ (no need to follow it strictly). It's always a good idea to have a python linter on the editor. I use `python-mode` plugin for vim.